packages/office-ui-fabric-react/src/components/ScrollablePane/examples/ScrollablePane.DetailsList.Example.tsx

```diff
@@ -136,7 +136,7 @@ export class ScrollablePaneDetailsListExample extends React.Component<
           position: 'relative'
         }}
       >
-        <ScrollablePane componentRef={this._scrollablePane} scrollbarVisibility={ScrollbarVisibility.always}>
+        <ScrollablePane componentRef={this._scrollablePane} scrollbarVisibility={ScrollbarVisibility.auto}>
```

Comment:
I see. Is it necessary for this bug fix though? I'd like to try to decouple the example change from the actual bug fix. cc: @mikewheaton if he has an opinion about the default state of the scrollbar for this `<Scrollpane/>` example.

Reply:
Actually this fix also fix the that the horizontal scroll bar was not scrolling sticky  footer( col total) thus I though of adding the fix for vertical scroll bar also. 

Repro steps:
resize the window of scrollpane example and scroll the horizontal scrollbar then the details header and row scrolls but footer does not scroll. 

I will update the description with this details too.

Reply:
I agree with changing it to `auto` so that we only see scroll bars when necessary. I don't know the full history here, but my guess would be that `always` was there to avoid having to reposition elements as the bar appears or disappears. But it looks like that's working now.